OptimizelySDK.Tests/OptimizelyConfigTests/OptimizelyConfigTest.cs

```diff
@@ -40,6 +42,62 @@ public void Setup()
 
         #region Test OptimizelyConfigService
 
+        static Type[] ParameterTypes = {
+                typeof(ProjectConfig),
+            };
+
+        private PrivateObject CreatePrivateOptimizelyConfigService(ProjectConfig projectConfig)
+        {
+            return new PrivateObject(typeof(OptimizelyConfigService), ParameterTypes,
+                    new object[]
+                    {
+                        projectConfig
+                    });
+        }
+
+        [Test]
+        public void TestGetOptimizelyConfigServiceSerializedAudiences()
+        {
+            var datafileProjectConfig = DatafileProjectConfig.Create(TestData.TypedAudienceDatafile, new NoOpLogger(), new ErrorHandler.NoOpErrorHandler());
+            var optlyConfigService = CreatePrivateOptimizelyConfigService(datafileProjectConfig);
+
+            var audienceConditions = new List<List<object>>
+            {
+                new List<object>() { "or", "3468206642", "3988293898" },
+                new List<object>() { "or", "3468206642", "3988293898", "3468206646" },
+                new List<object>() { "not", "3468206642" },
+                new List<object>() { "or", "3468206642" },
+                new List<object>() { "and", "3468206642" },
+                new List<object>() { "3468206642" },
+                new List<object>() { "3468206642", "3988293898" },
+                new List<object>() { "and", new JArray() { "or", "3468206642", "3988293898" }, "3468206646" },
+                new List<object>() { "and", new JArray() { "or", "3468206642", new JArray() { "and", "3988293898", "3468206646" } }, new JArray() { "and", "3988293899", new JArray() { "or", "3468206647", "3468206643" } } },
+                new List<object>() { "and", "and" },
+                new List<object>() { "not", new JArray() { "and", "3468206642", "3988293898" } }
```

Comment:
Let's add 2 more cases from TDD:
- empty audiences []
- not match id: ["or", "1", "999999999"]